runtime/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/stream/StdoutEventsStream.java

```diff
@@ -61,8 +62,9 @@ private void handleEvent(
     {
         final EventFW event = eventRO.wrap(buffer, index, index + length);
         String qname = context.supplyQName(event.namespacedId());
+        String eventName = context.supplyLocalName(event.id()).replace('.', '_').toUpperCase();
```

Comment:
Looks like this code is going to repeat in two ways.
 - allocate strings for the same event id as multiple events are logged
 - same code in different exporters

Given, that we want these event names to be consistent, we should move the implementation into the engine for reuse and handle the caching of event names there too.

In engine `EngineContext`, let's add
```
public String supplyEventName(
  int eventId);
```
Then in `EngineWorker` (which implements `EngineContext`.
```
private final Int2ObjectHashMap<String> eventNames;

...(initialize in constructor)...

@Override
public String supplyEventName(
  int eventId)
{
    return eventNames.computeIfAbsent(eventId, this::computeEventName);
}

private String computeEventName(
  int eventId)
{
    return supplyLocalName(eventId).replace('.', '_').toUpperCase();
}
```

Then adjust `StdoutExporterContext` to expose `supplyEventName(int eventId)` _instead_ of `supplyLocalName` (now unused), and the above code becomes:

```
String eventName = context.supplyEventName(event.id());
```